name: "Jetracer_decentralised"
decentralised: true
N: 20
integrator_step: 0.3
n_discs: 0
track_choice: "t_junction" # 'straight_line', sinus, "t_junction"
positive_track: false
add_slack: true

number_of_robots: 2
robot_1:
  start_x: -6.0
  start_y: 0.0
  start_theta: 0.0  # in radians
robot_2:
  start_x: 0.0
  start_y: -3.0
  start_theta: 0.5  # radians

verbose: false
debug_visuals: false
dart_simulator: false

solver_settings:
  iterations: 2 #2 # 1
  solver_type: SQP_RTI # SQP_RTI (default) or SQP

control_frequency: 10.0
braking_acceleration: 1.0

polytopic:
  d_min: 0.1
  length: 0.2
  width: 0.2

contouring:
  num_segments: 5
  use_huber: false

weights:
  throttle: .1
  steering: .1
  lambda: .1
  s_dual: .1
  dmin_objective: .1
  velocity: 0.4
  reference_velocity: 0.4
  lag: 0.6
  contour: 0.6
  terminal_angle: 1.0 #0.0
  terminal_contour: 5.0 # 0.0
  terminal_lag: 1.0 # 0.0






